thirdeye-ui/src/app/platform/utils/date-time/date-time.util.ts

```diff
@@ -129,12 +129,19 @@ export const formatDurationV1 = (
 // Returns formatted string representation of date and time
 // For example:
 // MMM DD, YYYY, HH:MM AM/PM
-export const formatDateAndTimeV1 = (date: number): string => {
+export const formatDateAndTimeV1 = (
+    date: number,
+    timezoneOverride?: string
```

Comment:
Just a suggestion, how about making the second parameter an object so it can support other optional params in the future if needed

`{
  timezoneOverride
}?: {
  timezoneOverride?: string
}`

Reply:
Ahh good suggestion. WIll make the change when we add more options